package proto_ff;

import "yd_fieldoptions.proto";


message E_MoneyrewardMoneyreward
{
	optional int32 m_taskreward = 1[(yd_fieldoptions.field_cname) = "任务奖励类型"];
	optional string m_boxtaskreward = 2[(yd_fieldoptions.field_cname) = "宝箱奖励", (yd_fieldoptions.field_bufsize) = 60];
	optional string m_boxnum = 3[(yd_fieldoptions.field_cname) = "宝箱奖励次数条件", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_task = 4[(yd_fieldoptions.field_cname) = "抽取的任务类型"];
	optional int32 m_taskmax = 5[(yd_fieldoptions.field_cname) = "任务上限（抽取次数）"];
	optional string m_oneprice = 6[(yd_fieldoptions.field_cname) = "一键完成单次货币价格", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_freeunlocklv = 7[(yd_fieldoptions.field_cname) = "免费一键开放等级"];
	optional int32 m_viplv = 8[(yd_fieldoptions.field_cname) = "双倍需求vip等级"];
	optional int32 m_timeget = 9[(yd_fieldoptions.field_cname) = "接取任务倒计时时间"];
	optional int32 m_timegetmoney = 10[(yd_fieldoptions.field_cname) = "领取倒计时时间"];
	optional int32 m_timego = 11[(yd_fieldoptions.field_cname) = "前往倒计时时间"];
}


message Sheet_MoneyrewardMoneyreward
{
	repeated E_MoneyrewardMoneyreward E_MoneyrewardMoneyreward_List = 1[(yd_fieldoptions.field_arysize)=20];
}
